crates/ruff_linter/resources/test/fixtures/ruff/suppressions.py

```diff
@@ -54,3 +54,35 @@ def f():
     # ruff:disable[E741,F841]
     I = 1  # noqa: E741,F841
     # ruff:enable[E741,F841]
+
+
+def f():
+    # TODO: Duplicate codes should be counted as duplicate, not unused
```

Comment:
It's fine if one of them counts as unused, for as long as we remove the right one.